examples/qfiae/utils_qfiae.py

```diff
@@ -0,0 +1,108 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from qibo import gates
+from qibo.models import Circuit
+from qibo.models.iqae import IQAE
+
+
+def fourier_series(coeffs, x_val, period=2 * np.pi):
+    """Computes the Fourier series for a given set of coefficients in exponencial form."""

```

Comment:
Recalling a discussion recently had in `Qibolab`, we should try to uniform the docstrings. 
Since we are referring to the [PEP-257 lineguides](https://peps.python.org/pep-0257/), we should write descriptions according to:

_The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”._

In a few words, please change all the third persons "Computes, ..." into imperative mood "Compute, ...".